src/oop/oop2.py

```diff
@@ -6,7 +6,8 @@
 class GroundVehicle():
     def __init__(self, num_wheels):
```

Comment:
Based on the instructions on line 3, num_wheels should default to 4 when not specified. One way of doing this would be 
def __init__(self, num_wheels=4):
    self.num_wheels = num_wheels